spec/code_climate.travis.rb

```diff
@@ -0,0 +1,4 @@
+require 'simplecov'
```

Comment:
we should also add this line to the travis ci: https://github.com/renuo/renuo-cms-api/blob/develop/.travis.yml#L19
also, you'll need to create a dummy file (which is empty), and to include it in the spec helper :)